lib/generator.js

```diff
@@ -38,7 +38,6 @@ var ProjectType = {
   wap: "WAP",
   website: "WEBSITE",
   node: "NODE",
-  ruby: "RUBY",
```

Comment:
Strange, why does Ruby come up in your diff? Maybe some conflict with @Hamster-Huey's change?